src/main/java/net/sacredlabyrinth/phaed/simpleclans/listeners/SCEntityListener.java

```diff
@@ -208,7 +208,7 @@ public void onPlayerInteract(PlayerInteractEntityEvent event)
                 {
                 	return;
                 }
-                if (cp.getClan().isMember((Player) tamed.getOwner()))
+                if (cp.getClan() != null && cp.getClan().isMember((Player) tamed.getOwner()))
```

Comment:
Did you test this fix?
getClan wouldn't return null on a ClanPlayer retrieved by getClanPlayer

Reply:
"cp" is probably the cause of the NPE

Reply:
Oh, sorry about that. I did test it, but not that thuroughly. It seemed like it was an intermittent issue, but aparently it only triggers if the player is not trusted on the grief prevention claim.

Reply:
> not trusted on the grief prevention claim.

GP probably cancels the event if they are trusted.

Anyway, the null check should be on "cp".